dev-requirements.txt

```diff
@@ -1,3 +1,4 @@
 pytest
 mypy
 pre-commit
+DALL-E==0.1
```

Comment:
Non block Q: is this temporary just to enable FLAVA? Feels odd to take a dep on openAI's discrete VAE since we are implementing our own (VQVAE).

Reply:
this comes due to the ckpt loading from dalle. so i don't have a good solution other than taking this dependency. longer term, yes we can hack the ckpt